README.md

```diff
@@ -31,7 +31,34 @@ A simple command that runs your program. For example,
 python <demo_name>.py
 ```
 
-If your example requires user input, make sure to specify any input limitations.
```

Comment:
Something to consider: now that people can come directly to one of our OS examples in GitHub and the repo provides the ability to just click it open in CodeSpaces (rather than the Leap IDE), the above Usage section should probably start with a line saying that a Leap account is needed.